src/lib/Provider/GoogleMapProvider.svelte.js

```diff
@@ -122,33 +123,37 @@ export default class GoogleMapProvider {
 
 		this.stopsMap.set(stop.id, stop);
 
-		marker.addListener('click', () => {
-			if (this.globalInfoWindow) {
-				this.globalInfoWindow.close();
-			}
+		marker.addListener('click', () => this.openStopMarker(stop));
 
-			if (this.popupContentComponent) {
-				unmount(this.popupContentComponent);
-			}
+		this.markersMap.set(stop.id, marker);
```

Comment:
Not 100% sure on this approach / if it's ok to add to this `markersMap` set, but I needed the Marker in order to be able to show it after clicking on the stop in the Route modal. LMK if this seems reasonable.

Reply:
I think this looks totally reasonable.